src/ccpp_fcall.F90

```diff
@@ -369,6 +369,7 @@ subroutine ccpp_run_scheme(scheme, cdata, ierr)
     end subroutine ccpp_run_scheme
 
 #if 0
+    ! DH 20180504 - keep for future use
```

Comment:
Please add detail on when and how this code might be used so other developers know what this is so this does not become a ghost block.

Reply:
I guess people can use the date in my comment to infer some time later that the routine really is not needed anymore (could be myself even).